src/commands/hop.tsx

```diff
@@ -43,12 +43,17 @@ function Hop() {
 	};
 
 	const items = useMemo(() => {
-		return allBranches
-			.filter(branch => branch !== currentBranch)
-			.map(branch => ({
-				label: branch,
-				value: branch,
-			}));
+		const shouldFilter = allBranches.length > 1;
+		let maybeFilteredAllBranches = allBranches;
+		if (shouldFilter) {
+			maybeFilteredAllBranches = allBranches.filter(
+				branch => branch !== currentBranch,
+			);
+		}
+		return maybeFilteredAllBranches.map(branch => ({
+			label: branch,
+			value: branch,
+		}));
```

Comment:
Should we return early with the exception case? And then potentially document why we need to return early there?

Reply:
Or potentially having the check inside the filter call?

So something like:
```js
includeCurrentBranch = allBranches.length === 1
allBranches.filter(branch => includeCurrentBranch || branch !== currentBranch).map(...)
```